systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -462,6 +465,63 @@ public void prepareEnvForOperator(ExtensionContext extensionContext, String clie
         prepareEnvForOperator(extensionContext, clientNamespace, Collections.singletonList(clientNamespace));
     }
 
+    private String changeLeaseNameInResourceIfNeeded(String yamlPath) {
+        final EnvVar leaseEnvVar = extraEnvVars.stream().filter(envVar -> envVar.getName().equals("STRIMZI_LEADER_ELECTION_LEASE_NAME")).findFirst().orElse(null);
+        if (leaseEnvVar != null && yamlPath.contains("022-")) {
```

Comment:
I am little bit sceptical about that condition (i.e., yamlPath.contains("022-")) but something I can live with :D Maybe being explicit in this case would be better. We don't know if another file with 022- prefix will be added in the future, which could potentially break this. But it is just my though...

Reply:
Yeah I know, as @Frawless mentioned I'll create a list of resource with proper test path rather than this :) good point!